### PR TITLE
doc: matter: Fixes for Matter samples

### PR DESCRIPTION
**Pull request by ArekBalysNordic**
- Fixed missing WiFi configuration for nR54H20 in docs.
- Fixed HW model v2 for nrf54l15 in tempalte.

### CODE REVIEW

**Review by doublemis1 — APPROVED** [review]

**Review by Damian-Nordic — APPROVED** [review]

**Comment by shanthanordic** [comment]
@ArekBalysNordic  techwriter review missing

**Comment by ArekBalysNordic** [comment]
> @ArekBalysNordic techwriter review missing 

@wiba-nordic is assigned on behalf of https://github.com/orgs/nrfconnect/teams/ncs-matter.

File: samples/matter/lock/README.rst
[the remark this answers is not in the document]
Comment (ArekBalysNordic) [reply samples/matter/lock/README.rst:1]: WiFi-Thread switched variant is not available on nRF54H20 yet, should I point this somewhere, or just leave all mentioned places regarding that unedited?

[the remark this answers is not in the document]
Comment (wiba-nordic) [reply samples/matter/lock/README.rst:1]: No wait, support is only for Matter over Wi-Fi, is it? Not for switching? Then most of these can be ignored.

[the remark this answers is not in the document]
Comment (ArekBalysNordic) [reply samples/matter/lock/README.rst:1]: Yeah, that's right, for now only a simple Matter over WiFi on nRf54H20.

**Review by wiba-nordic — APPROVED** [review]

**Pull request merged**